src/radical/pilot/configs/resource_local.json

```diff
@@ -24,7 +24,8 @@
         "mpi_launch_method"           : "MPIEXEC",
         "rp_version"                  : "debug",
         "virtenv"                     : "%(global_sandbox)s/ve_localhost",
-        "virtenv_mode"                : "create"
+        "virtenv_mode"                : "create",
+        "cores_per_node"              : 1
```

Comment:
Ah, right,  sorry for that...  I'll undo (even if I find it useful, as that is the only way to have non-trivial agent configs on localhost.  But I should put that into my private configs, apologies!)